### DISCUSSION TITLE
Installation and usage instructions

### DISCUSSION THREAD

**Discussion by tsaijamey** [Draym/discord-puppet#2] [Q&A] [answered]
Although I haven't started using your project yet, I'd like to thank you in advance. Besides, I'm not familiar with Node, and I only know simple package installation commands. However, I'm unfamiliar with how to package your TypeScript files. I hope you can provide more detailed installation and usage instructions. Thanks again.

**Comment by Draym** [comment, marked as answer]
Hi @tsaijamey, can you give me more details on how you hope to use this project for ?

If you are trying to run discord commands or interact with midjourney bot in a nodejs application, then you are on the right repo.
If you already have a nodejs project, all you have to do is to install the dependency:
```bash
npm i @d-lab/discord-puppet 
```
Then you can also checkout this example file on how to run commands on midjourney: https://github.com/Draym/discord-puppet/blob/main/bin/midjourney-cmds.ts
